src/Adapter/Cart/CommandHandler/UpdateProductQuantityInCartHandler.php

```diff
@@ -194,6 +195,16 @@ private function assertProductIsInStock(Product $product, UpdateProductQuantityI
             return;
         }
 
+        if (Pack::isPack($product->id)) {
```

Comment:
```suggestion
        elseif (Pack::isPack($product->id)) {
```

Suggestion, so we distinguish both behaviour more easily (a product can't be a pack and have combinations)